src/Behat/Behat/HelperContainer/BuiltInServiceContainer.php

```diff
@@ -12,7 +12,7 @@
 
 use Behat\Behat\HelperContainer\Exception\ServiceNotFoundException;
 use Behat\Behat\HelperContainer\Exception\WrongServicesConfigurationException;
-use Interop\Container\ContainerInterface;
+use Psr\Container\ContainerInterface;
```

Comment:
our class must keep implementing the interop interface to be BC (which will make it implement the PSR one)